src/components/layout/vertical-nav.tsx

```diff
@@ -43,32 +43,41 @@ const NavLink: FC<
   );
 };
 
+const companies = [
+  {
+    name: "Work Go",
+    domain: "app.workgo.com",
+    logo: "/logo.png",
+  },
+  {
+    name: "Google",
+    domain: "google.com",
+    logo: "/google.png",
+  },
+  {
+    name: "Facebook",
+    domain: "facebook.com",
+    logo: "/facebook.png",
+  },
+  {
+    name: "Amazon",
+    domain: "amazon.com",
+    logo: "/amazon.png",
+  },
+];
+
 export default function VerticalNav() {
+  const [selectedCompany, setSelectedCompany] = useState(companies[0]);
   return (
     <div className="h-screen w-[300px] fixed bg-[#161618] text-white overflow-auto flex flex-col  p-3 gap-3">
-      <div className="flex items-center justify-start gap-2 px-6 py-3 rounded-lg hover:bg-[#101012] cursor-pointer transition-colors group">
-        <img src="/logo.png" alt="Work Go" className="w-14 h-14" />
-        <div className="w-full flex justify-between gap-2">
-          <div className="space-y-1.5">
-            <h1 className="font-bold text-2xl leading-4">Work Go</h1>
-            <p className="text-gray-100 group-hover:text-white transition-colors tracking-widest font-extralight text-xs">
-              app.workgo.com
-            </p>
-          </div>
-
-          <button
-            type="button"
-            className="text-gray-100 group-hover:text-[#99ff00] transition-colors shrink-0"
-          >
-            <ChevronDown className="w-4 h-4" />
-          </button>
-        </div>
+      <div>
+        <CompanyUi
+          {...selectedCompany}
```

Comment:
I think it would be much more readable if we pass the selectedCompany as a single prop

Reply:
Done